kwonhayeon/SpringPrac/src/main/java/com/likelion/springprac/domain/User.java

```diff
@@ -0,0 +1,15 @@
+package com.likelion.springprac.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data // getter, setter, toString 등 자동 생성
+@AllArgsConstructor // 모든 필드를 받는 생성자
+@NoArgsConstructor // 기본 생성자
+
```

Comment:
`@Entity`를 사용하여 이 클래스가 JPA에서 관리할 엔티티 클래스임을 명시해주어야해요!
```suggestion
@Entity
```